### ISSUE TITLE
Vertical Splitter in Edge and Firefox (Vaadin 10)

### ISSUE THREAD

**Issue by pleku** [vaadin/vaadin-split-layout#125]
@OlgaIOn commented on [Mon Nov 26 2018](https://github.com/vaadin/flow/issues/4816)

Vertical Splitter is displayed incorrectly in Edge and Firefox:

(![verticalsplitter](https://user-images.githubusercontent.com/45284497/49024716-71204d00-f19a-11e8-8cec-36f01e169d98.png))

---

@pleku commented on [Tue Nov 27 2018](https://github.com/vaadin/flow/issues/4816#issuecomment-441941997)

Hi @OlgaIOn . Is this the `SplitLayout` component ? 

I tested the demo on https://vaadin.com/components/vaadin-split-layout/java-examples with FF and Edge on Windows 10, and I don't see anything broken there, so could you please add a minimal test case to reproduce this ?

---

@OlgaIOn commented on [Tue Nov 27 2018](https://github.com/vaadin/flow/issues/4816#issuecomment-441964815)

Hi @pleku . Yes, it is SplitterLayout. 
Primary split contains Grid, secondary split contains TextArea, SplitterPosition - 50.0

Browser:
-  Firefox Quantum (Linux Mint)
-  Edge (Windows 10)

**Comment by limonte** [comment]
> Vertical Splitter is displayed incorrectly in Edge and Firefox:

On the screenshot, there's horizontal, not a vertical splitter.

> Primary split contains Grid, secondary split contains TextArea, SplitterPosition - 50.0

Vertical layout with grid and textarea works nicely:

- live demo: https://equal-growth.glitch.me/
- source code: https://glitch.com/edit/#!/equal-growth?path=index.html:60:59

@pleku could you please create a glitch with the live demo of the issue?

**Comment by OlgaIOn** [comment]
> On the screenshot, there's horizontal, not a vertical splitter.

Yes, it is horizontal splitter, but looks like vertical. 

We create SplitLayout (by default horizontal, as far as I see), set primary and secondary split and splitter position. Nothing more

`SplitLayout layout = new SplitLayout();`
` layout.addToPrimary(buildGrid());`
` layout.addToSecondary(buildTextEdit());`
`layout.setSplitterPosition(50.0);`

**Comment by limonte** [comment]
Here's the live demo with horizontal layout, grid, and textarea: https://alert-currency.glitch.me/

@pleku I don't think the issue is on the front-end part.

**Comment by pleku** [comment]
@limonte I agree, but I was not able to reproduce it with the Flow parts either.

@OlgaIOn is there any custom CSS used or how are you wrapping the split layout, is it inside another layout ?

**Comment by OlgaIOn** [comment]
In Firefox on Windows is all right. It does not work in
-     Firefox Quantum (Linux Mint)
-     Edge (Windows 10)

We did not use custom CSS. Split layout is inside Tab

**Comment by pleku** [comment]
>We did not use custom CSS. Split layout is inside Tab

I'm not sure if I understand correctly that you have the `SplitLayout` inside `Tabs` / `Tab` ? (https://vaadin.com/components/vaadin-tabs/java-examples)
That makes no sense, since the content for each `Tab` should be shown outside the `Tabs` itself.

**Comment by OlgaIOn** [comment]
> > We did not use custom CSS. Split layout is inside Tab
> 
> I'm not sure if I understand correctly that you have the `SplitLayout` inside `Tabs` / `Tab` ? (https://vaadin.com/components/vaadin-tabs/java-examples)
> That makes no sense, since the content for each `Tab` should be shown outside the `Tabs` itself.

Our SplitLayout is a content for Tab.  

I would like to emphasize again, the problem occurs in
     Firefox Quantum (Linux Mint)
     Edge (Windows 10)

Everything is ok in Chrome (both Linux and Windows), Mozilla (Windows). So it seems to be a cross browser compatability problem.
